# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Surface.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/Surface.h
--- a/gfx/angle/checkout/src/libANGLE/Surface.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/Surface.h	Sat Sep 07 13:38:36 2019 +0000
@@ -68,6 +68,8 @@
     EGLint getType() const;
 
     Error initialize(const Display *display);
+    Error makeCurrent(const gl::Context *context);
+    Error unMakeCurrent(const gl::Context *context);
     Error swap(const gl::Context *context);
     Error swapWithDamage(const gl::Context *context, EGLint *rects, EGLint n_rects);
     Error postSubBuffer(const gl::Context *context,
@@ -85,7 +87,6 @@
     EGLint isPostSubBufferSupported() const;
 
     void setSwapInterval(EGLint interval);
-    Error setIsCurrent(const gl::Context *context, bool isCurrent);
     Error onDestroy(const Display *display);
 
     void setMipmapLevel(EGLint level);
@@ -226,7 +227,7 @@
   private:
     Error destroyImpl(const Display *display);
 
-    void postSwap(const gl::Context *context);
+    void postSwap(const Display *display);
     Error releaseRef(const Display *display);
 
     gl::InitState mInitState;